grid2op/MakeEnv/Make.py

```diff
@@ -154,6 +159,17 @@ def _extract_ds_name(dataset_path):
     dataset_name = os.path.splitext(dataset_name)[0]
     return dataset_name
 
+def _aux_is_multimix(dataset):
+    ds_name = os.path.basename(dataset)
+    if ds_name in MULTIMIX_ENVS:
```

Comment:
not a fan at all of this. This re introduce a coupling between the env release and the dataset release

Reply:
Solved by using the meta file solution.